app/graph/types/asset_type.rb

```diff
@@ -9,6 +9,7 @@ module Types
       field :id, !types.ID, 'Uniq ID for this asset'
```

Comment:
One meta comment -- since we're doing some refactoring here, can we rename the folder `graph` to `graphql`? That way it matches the `graphql` folder in `specs` while also favoring the literal over the hip 😎 (No strong opinion about this, though, just something I'd rather see as a hypothetical new arrival to the codebase.)

Reply:
Sure, no strong opinion either but seems harmless!